opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/dto/result/body/AuthStateBody.java

```diff
@@ -38,6 +39,11 @@
      */
     private AuthResultBody resultBody;
 
+    /**
+     * Challange data, needed for embedded sca
```

Comment:
`Challange data, needed for embedded sca` ->
`Challange data, needed for embedded SCA for OpticTAN, PhotoTAN and other challeges that require some data to be shown to user`

Reply:
text was changed